supervision/tracker/ByteTrack/config/byte_track.yaml

```diff
@@ -0,0 +1,5 @@
+byte_track:
```

Comment:
Our ByteTrack implementation should be configurable through the constructor arguments. Let's get rid of this `yaml` and `config` directory.